XamarinCommunityToolkitSample.sln

```diff
@@ -36,81 +36,141 @@ Global
 		{8B80BFA6-7B19-4DE7-BD97-7D84194AD0C2}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
 		{8B80BFA6-7B19-4DE7-BD97-7D84194AD0C2}.Debug|Any CPU.Build.0 = Debug|Any CPU
 		{8B80BFA6-7B19-4DE7-BD97-7D84194AD0C2}.Debug|Any CPU.Deploy.0 = Debug|Any CPU
+		{8B80BFA6-7B19-4DE7-BD97-7D84194AD0C2}.Debug|ARM.ActiveCfg = Debug|Any CPU
```

Comment:
Don't think this is useful and a difference between some IDE versions, I don't really care, but since you're making another change could you undo this? Thanks!

Reply:
Consider it done 🙂

Reply:
You're my hero